app/assets/stylesheets/application/_variables.scss

```diff
@@ -1,44 +1,47 @@
-// Custom variables
-$red: #cc0000;
-$darkred: #950002;
-$darkgrey: #353535;
-$lightgrey: #ddd;
-$mediumgrey: #636363;
+$base-font-size:   16px;
+$small-font-size:  $base-font-size * 0.875;
+$base-line-height: 1.5;
+
+/* colors */
+$eff-red: #ec1e1e;
+$red: #FF0000;
```

Comment:
White against `#FF0000` is pretty harsh to read IMO. It fails WCAG AA for normal text. The shade we're currently using (`#CC0000`) is better, but contrasts poorly with `#949494` the tool background color (and fails the WCAG check for UI components). Not sure what to suggest here -- just flagging it as a drawback.

Reply:
Thanks, Peter, I noticed this is well and will discuss with design.